index.html

```diff
@@ -110,7 +110,7 @@ <h2>Principles</h2>
   <dd><p>When we are adding new web technologies and platforms, we will build them to cross regional and national boundaries. People in one location should be able to view web pages from anywhere that is connected to the web.</p></dd>
 
   <dt>The web should not cause harm to society</dt>
-  <dd><p>When we are adding a feature or technology to the web, we will consider what harm it could do to society or groups, especially vulnerable people. We will prioritise potential benefits for users over potential benefits to web developers, in line with the <a href="https://www.w3.org/TR/html-design-principles/#priority-of-constituencies">priority of constituencies</a> from the HTML Design Principles.</p></dd>
+  <dd><p>When we are adding a feature or technology to the web, we will consider what harm it could do to society or groups, especially vulnerable people. We will prioritise potential benefits for users over potential benefits to web developers, in line with the <a href="https://www.w3.org/TR/html-design-principles/#priority-of-constituencies">priority of constituencies</a> from the HTML Design Principles. We will build new web technologies in an open and collaborative matter according to open processes (for example, the <a href="https://www.w3.org/Consortium/Process/"></a>W3C process</a>). Furthermore, by adhereing to codes of conduct (such as the W3C <a href="https://www.w3.org/Consortium/cepc/"></a>code of ethics and professional conduct</a>) we will ensure the requirements and views of marganlized communities are respected.</p></dd>
```

Comment:
adhering

Reply:
I note the inconsistent use of en-gb and en-us at least in this section. e.g. "prioritise" / "marginalized"

Reply:
fixed